docs/src/components/landing/Hero.js

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import IconImage from 'docs/src/components/icon/IconImage';
+import GradientText from './GradientText';
```

Comment:
Why not import from?

```suggestion
import GradientText from 'docs/src/components/typography/GradientText';
```

The dark mode look different.